browser/src/map/handler/Map.FileInserter.js

```diff
@@ -140,8 +188,15 @@ L.Map.FileInserter = L.Handler.extend({
 					for (var i = 0; i < byteBuffer.length; i++) {
 						strBytes += String.fromCharCode(byteBuffer[i]);
 					}
-					window.postMobileMessage('insertfile name=' + aFile.name + ' type=' + type +
-										       ' data=' + window.btoa(strBytes));
+
+					if (type === 'multimedia') {
+						window.postMobileMessage('insertfile name=' + aFile.name + ' type=' + type +
+											       ' data=' + window.btoa(strBytes) +
```

Comment:
hmm, might be a big payload. Are there any limitations for a size?
isn't there any option for sending HTTP POST request to the "server" in the mobile ?

Reply:
I understand. My concern is: previously we had image now video/media which might be significantly bigger in average case. HTTP transport is used for clipboard which also could have bigger content. I see it as more reliable in such cases as there is always some streaming support built-in